common/options/options_test.go

```diff
@@ -422,3 +426,37 @@ func TestHiddenOptionsDefaults(t *testing.T) {
 	})
 
 }
+
+func TestDeprecationWarning(t *testing.T) {
+	if !(testtype.HasTestType(testtype.SSLTestType) && testtype.HasTestType(testtype.SSLTestType)) {
```

Comment:
Why is this condition `testtype.HasTestType(testtype.SSLTestType)` duplicated here?

Reply:
ah, no idea why I have them twice. Thanks for pointing out!